### PR TITLE
chore: add info package to webpack-cli

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
Integrate info package into the `webpack-cli` by default.

**Did you add tests for your changes?**
N/A most probably

**If relevant, did you update the documentation?**
Needs to be updated (if any)

**Summary**
This is part of roadmap of `webpack-cli`.

**Does this PR introduce a breaking change?**
No

**Other information**
#717

### CODE REVIEW

File: packages/webpack-cli/__tests__/info.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const path = require('path');
+
+describe('Info', () => {
+    it('should run with cli', () => {
+        const { stdout, stderr } = spawnSync(path.resolve(__dirname, '../bin/cli.js'), [], {
+            cwd: path.resolve(__dirname),
+            reject: false,
+        });
+        expect(stdout).toBeTruthy();
+        expect(stderr).toBeFalsy();
```
Comment (evilebottnawi) [inline packages/webpack-cli/__tests__/info.test.js:11]: Can we tests that current output contains information from `info` pcakge, i.e. check what we have fields - `Binaries`/`Browsers`/`System`/`npmGlobalPackages`/`npmPackages`

Comment (rishabh3112) [reply packages/webpack-cli/__tests__/info.test.js:11]: It's a integration test, info package as a whole is already tested.

Comment (rishabh3112) [reply packages/webpack-cli/__tests__/info.test.js:11]: check `test/info` directory.

Comment (evilebottnawi) [reply packages/webpack-cli/__tests__/info.test.js:11]: Please improve tests, we can do regression(s) and `webpack info` can potential output any information (for example output version of package) and tests only on not empty output is not enough

Comment (rishabh3112) [reply packages/webpack-cli/__tests__/info.test.js:11]: Okay I got it now, adding them my bad!

**Comment by rishabh3112** [comment]
Yes, it may be outputting anything (like webpack output, or prompting to install info, etc)

**Comment by evilebottnawi** [comment]
@rishabh3112 With each PR you become better and better developer, congrats!

File: packages/webpack-cli/__tests__/info.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        expect(stdout).toContain('OS');
+        expect(stderr).toBeFalsy();
+    });
+
+    it('should work with flags', () => {
+        const { stdout, stderr } = spawnSync(path.resolve(__dirname, '../bin/cli.js'), ['info', '--output=json'], {
+            cwd: path.resolve(__dirname),
+            reject: false,
+        });
+        console.log(stdout);
```
Comment (jamesgeorge007) [inline packages/webpack-cli/__tests__/info.test.js:21]: Unnecessary log.

Comment (rishabh3112) [reply packages/webpack-cli/__tests__/info.test.js:21]: Thanks

**Comment by evilebottnawi** [comment]
@rishabh3112 Oh, we used invalid commit message, it is feature, not chore, please do not rush when we are merging in the future

**Comment by rishabh3112** [comment]
Sorry for that, I merged after approvals though. Info was already there, How is this PR a feature?

**Comment by evilebottnawi** [comment]
Feature: the `info` command is part of `webpack-cli` package, no need install that

**Pull request merged**